plugins/package-managers/cocoapods/src/funTest/kotlin/CocoaPodsFunTest.kt

```diff
@@ -74,5 +78,29 @@ class CocoaPodsFunTest : WordSpec({
 
             result.withInvariantIssues().toYaml() should matchExpectedResult(expectedResultFile, definitionFile)
         }
+
+        "determine dependencies from a Podfile located in a subdirectory referring to npm packages" {
+            // The NPM package manager is run first to initialize the node_modules directory, which is required to
+            // parse the Podfile successfully.
+
+            val npmDefinitionFile = getAssetFile("projects/synthetic/subdirectory/package.json")
+            val result = analyze(
+                npmDefinitionFile.parentFile,
+                true,
+                packageManagers = listOf(NpmFactory(), CocoaPodsFactory()),
+                packageManagerConfiguration = mapOf("CocoaPods" to PackageManagerConfiguration(listOf("NPM")))
```

Comment:
Let's used named parameters with `PackageManagerConfiguration` so that it's more readable.

Reply:
While at it, you should do the same esp. for the boolean argument in line 89.